vmware/docker-volume-vsphere
# Revert "Adding photon known issue reference [SKIP CI]"

Reverts vmware/docker-volume-vsphere#1223

I've verified the customer scenario (mentioned at https://github.com/vmware/docker-volume-vsphere/issues/1215#issue-225865211) by taking latest Photon OVA as per https://github.com/vmware/photon/issues/640#issuecomment-299588168

Plugin installation works fine as expected. 

Testing done:
1) install plugin > docker plugin ls
2) create volume `-d vsphere` > verify volume creation by `docker volume ls` & `docker volume inspect`

Reverting document change.

/CC @pdhamdhere
